content/_guides/managing-user-roles-in-a-tenant.md

```diff
@@ -57,8 +50,8 @@ drop-down menu.
 As a workspace Owner and Admin you can set the user role while inviting them to
 the Contract and Workspace.
 
-> **Please Note** You can invite to Contract and Workspace only if your have
-> **Owner** or **Admin** role in the contract.
+> **Please Note** You can invite to Contract and Workspace you have either **Owner**
+> or **Admin** role in the contract.
```

Comment:
Contract Admin doesn't have permissions to add/invite new members to the Contract

Reply:
The same in this sections

> As a contract **Owner** or **Admin** you can modify roles of any user in the
contract by visiting **Members** section of the **Contract Settings** and
clicking to edit (the pencil).

Reply:
`{
                    "i18n": {
                        "en": "Admin"
                    },
                    "role": "admin",
                    "permissions": [
                        "contracts.workspace.create",
                        "contracts.workspace.listAll",
                        "contracts.workspace.delete",
                        "contracts.repository.edit",
                        "contracts.devTeam.edit",
                        "global.auth_clients.create",
                        "global.auth_clients.get",
                        "global.auth_clients.edit",
                        "global.auth_clients.delete"
                    ],
                    "scope": "contracts"
                },`

Reply:
And remove pls Admin from

> As a contract **Owner** or **Admin** you can modify roles of any user in the
contract by visiting **Members** section of the **Contract Settings** and
clicking to edit (the pencil).